src/main/scala/beam/router/BeamSkimmer.scala

```diff
@@ -422,14 +491,56 @@ class BeamSkimmer @Inject()(val beamConfig: BeamConfig) extends IterationEndsLis
                       )
                     }
                   }
+
+                if (trie.contains(Path(origin.tazId, destination.tazId, timeBin))) {
```

Comment:
It can be simplified by something like:
```scala
val key = Path(origin.tazId, destination.tazId, timeBin)
trie.get(key).foreach { timeObserved =>
  series.add(theSkim.time,  timeObserved)
  writerObservedVsSimulated.write(
    s"${origin.tazId},${destination.tazId},${timeBin},${theSkim.time},$timeObserved\n"
  )
}
```